### ISSUE TITLE
Chinese display problem in jupyter notebook

### ISSUE THREAD

**Issue by secsilm** [ufoym/deepo#69]
I use the following code to download and start my jupyter notebook:

```bash
docker pull ufoym/deepo:all-py36-jupyter
docker run -it -p 8888:8888 --ipc=host ufoym/deepo:all-py36-jupyter jupyter notebook --no-browser --ip=0.0.0.0 --allow-root --NotebookApp.token= --notebook-dir='/root'
```

But there is a Chinese display problem in jupyter notebook:

![](https://i.imgur.com/VCYpUod.png)

Is there any suggestion? Thanks a lot.

**Comment by lihaoHuawei** [comment]
the same to my problem.
I can‘t rename notebook in Chinese format.
![image](https://user-images.githubusercontent.com/24602093/55299080-d5625e80-5463-11e9-8228-a65d67bf5a87.png)

Is the locale problem?
![image](https://user-images.githubusercontent.com/24602093/55299129-1195bf00-5464-11e9-8f85-b628334d73b9.png)

**Comment by lihaoHuawei** [comment]
@ufoym

**Comment by lihaoHuawei** [comment]
原因是此镜像的Ubuntu系统不支持中文。
解决办法：
1.安装中文包
2.设置locale
我重新制作了一个新镜像
```shell
FROM ufoym/deepo:all-py36-jupyter-cpu
RUN apt-get update
RUN apt-get install -y language-pack-zh-hans language-pack-zh-hans-base
ENV LANG zh_CN.UTF-8
```

```shell
docker build -t ufoym/deepo:all-py36-jupyter-cpu-zh .
```

可以重命名以及打开中文文件

**Comment by ufoym** [comment]
Fixed by 94cbd155ccdce6da8cab2d005f28f0305f520126
